### PR TITLE
move to new sim extension

### PR DESCRIPTION
**Pull request by tballmsft**

### CODE REVIEW

File: targetconfig.json [hunk 1]
```diff
@@ -478,6 +478,12 @@
                     "devUrl": "http://localhost:5173",
                     "aspectRatio": 3.45
                 }
+            },
+            "microbit-apps/pxt-arcadeshield": {
+                "simx": {
+                    "sha": "4055519e34baee00656d83b963dc1ff3a104afa2",
```
Comment (eanders-ms) [inline targetconfig.json:484]: I believe this is the sha you want, the most recent publish to ghpages:
```suggestion
                    "sha": "98c47e4eaeceeb6bf0e26099743df2fdd2b5dd50",
```

Comment (eanders-ms) [reply targetconfig.json:484]: The steps are outlined here: https://github.com/microsoft/pxt-simx-sample/tree/master/simx#releasing-your-simulator-extension. I probably should move this doc to pxt repo for discoverability.

Comment (tballmsft) [reply targetconfig.json:484]: I am confused about which commit hash to use. It should be the one associated to the gh-pages deploy?

Comment (tballmsft) [reply targetconfig.json:484]: how do I coordinate release of extension? Separate issue?

Comment (eanders-ms) [reply targetconfig.json]: Changes to targetconfig.json go live upon PR merge, or pretty soon thereafter.

Comment (eanders-ms) [reply targetconfig.json]: Please let me know if the doc linked above doesn't clearly indicated which hash to use. I've tried to ensure it communicates this as clearly as possible. If it isn't there yet, I'd like to address it.

**Review by eanders-ms — APPROVED** [review]

**Pull request merged**